tests/prob1-tests.rb

```diff
@@ -0,0 +1,34 @@
+require 'test/unit'
+require_relative "../problems/prob1"
+
+class UnitTests < Test::Unit::TestCase
```

Comment:
Tests look very nice, they have proper structure and they mostly test the right thing, a noticeable exception is that nil check.